client/boot/project/wordpress-com.js

```diff
@@ -32,19 +32,21 @@ const config = require( 'config' ),
 import { getSelectedSiteId, getSectionName } from 'state/ui/selectors';
 import { setNextLayoutFocus, activateNextLayoutFocus } from 'state/ui/layout-focus/actions';
 
-function renderLayout( reduxStore ) {
-	const Layout = require( 'controller' ).ReduxWrappedLayout;
+function renderLayoutPromise( reduxStore ) {
+	return require( 'controller' ).getReduxWrappedLayout( reduxStore ).then(
```

Comment:
Thanks for the research, @alexsanford, it’s a rabbit hole, indeed. I guess it’s one more item to be punted for until webpack2 support gets in.